package.json

```diff
@@ -53,14 +52,16 @@
     "winston-daily-rotate-file": "^4.5.1"
   },
   "devDependencies": {
+    "@tsconfig/node18": "^2.0.0",
     "@types/cors": "^2.8.12",
     "@types/express": "^4.17.14",
     "@types/extend": "^3.0.1",
     "@types/jasmine": "^4.0.3",
     "@types/js-yaml": "^4.0.0",
     "@types/nedb": "^1.8.11",
-    "@types/node": "^16",
+    "@types/node": "^18",
```

Comment:
Likewise, `package.json` should have its `"engines"` section updated to `"node": ">=18"` like it was for [matrix-appservice-node](https://github.com/matrix-org/matrix-appservice-node/commit/3413960453c63f16e79de95c1979ff5d25b71d6f#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R8).